usaspending_api/etl/es_transaction_template.json

```diff
@@ -669,6 +669,12 @@
             "type": "keyword"
           }
         }
+      },
+      "program_activity_names": {
```

Comment:
Similar question as the awards template

Reply:
Adding the `text` version to be consistent with other text fields.